package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "magento2",
+  "version": "1.0.0",
```

Comment:
is this right?

Reply:
nope, defaulted by the init 👍

Reply:
not harmful either though

Reply:
fine, only noticed cause we've had some version bump mixups in the recent past

Reply:
Will remove